src/pages/CDP.js

```diff
@@ -114,21 +117,59 @@ const CdpViewCard = ({ title, rows, isAction }) => {
   );
 };
 
-function CDPView({ cdpId }) {
+function CDPView({ cdpTypeSlug, getIlk }) {
   const { maker } = useMaker();
 
   // TODO cdpTypeSlug should become `id` or we should have both cdpTypeSlug AND id.
   const [cdpState, setCDPState] = useState(null);
+  const [collateralVal, setCollateralVal] = useState('');
+  const [debtVal, setDebtVal] = useState('');
+  const [ilk, setIlk] = useState(null);
 
   useEffect(() => {
     (async () => {
       const cdpManager = maker.service('mcd:cdpManager');
-      const cdp = await cdpManager.getCdp(cdpId);
+      const cdp = await cdpManager.getCdp(parseInt(cdpTypeSlug));
       setCDPState(cdp);
+      setIlk(getIlk(cdp.ilk));
+      setCollateralVal(await cdp.getCollateralValue());
+      setDebtVal(await cdp.getDebtValue());
     })();
   }, [cdpId, maker]);
```

Comment:
We might want to pass `[]` instead of `[cdpId, maker]` here so that this behaves like `componentDidMount`,  but I guess we'd need to understand more about how navi handles navigations from one cdp page to another.